Source/DafnyDriver/DafnyDocHtml.cs

```diff
@@ -0,0 +1,315 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.IO;
+using System.Linq;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Text;
+
+namespace Microsoft.Dafny;
+
+class DafnyDocHtml {
+
+  public static readonly string eol = "\n";
+  public static readonly string br = "<br>";
+  public static readonly string spaceTab = "&nbsp;  ";
+  public static readonly string mdash = " &mdash; ";
+  public static readonly string space4 = "&nbsp;&nbsp;&nbsp;&nbsp;";
+  public static readonly string initialbar = Code("&nbsp;&nbsp;|");
+
+  public static string HtmlStart(string title, string script = "") {
+    return
+      @"
+<!doctype html>
```

Comment:
```suggestion
<!DOCTYPE html>
```
It's not compulsory as it's case insensitive, but the majority of HTML pages I saw had this uppercase.